docs/section19/19-10.md

```diff
@@ -0,0 +1,73 @@
+                - Case 3: Elif the system primarily serves computer rooms and the baseline system type is 3 or 4 and an economizer is modeled then fail (this is exception #3): `elif (hvac.fan_system.AirEconomizer != Null AND hvac.fan_system.AirEconomizer.type == "TEMPERATURE") AND hvac_sys_type_b in ["SYS-3", "SYS-3a", "SYS-3b", "SYS-3c", "SYS-4"] AND hvac.id in HVAC_systems_primarily_serving_comp_rooms_list: outcome = "FAIL" and raise_message "This system [hvac.id] appears to meet the criteria associated with Section G3.1.2.6 exception #3 which is that an economizer shall not be modeled in the baseline for systems that serve computer rooms complying with Section G3.1.2.6.1. "`  
+                - Case 4: Elif the building has refrigerated cases, the building segment building area type is retail and an economizer is modeled in the baseline but not the proposed and so the system likely meets exception #2 so outcome is UNDETERMINED (this is exception #2): `elif (hvac.fan_system.AirEconomizer != Null AND hvac.fan_system.AirEconomizer.type == "TEMPERATURE") AND hvac in hvac_system_exception_2_list: outcome = "UNDETERMINED" and raise_message "This system [hvac.id] appears to meet the criteria associated with Section G3.1.2.6 exception #2 which is that an economizer shall not be modeled in the baseline for systems where the use of outdoor air for cooling will affect supermarket open refrigerated case-work systems and the proposed system does not include an economizer. An economizer has been modeled in the baseline when it appears this exception may apply. Manual check recommended."`                     
+                - Case 5: Elif there is an AirEconomizer object associated with the fan and the AirEconomizer type is dry bulb (i.e., Temperature) then pass: `elif hvac.fan_system.AirEconomizer != Null AND hvac.fan_system.AirEconomizer.type == "TEMPERATURE": outcome = "PASS"`  
+                - Case 6: Else, fail: `Else: outcome = "Fail and raise_message "Fail unless the system [include hvac.id] includes a gas-phase air cleaning to meet the requirements of Standard 62.1, Section 6.1.2. This, G3.1.2.6 exception #1, shall be used only if the system in the proposed design does not match the building design."`  
+
+**Notes/Questions:**  
+1. The outcome could be NOT_APPLICABLE if the HVAC baseline system was not defined correctly. I guess this is fine because they will have to fix the HVAC system anyway and this check will just have to be run at the next iteration.   
+2. With exceptions 2 and 3, shall we let it pass if an economizer is modeled in the baseline since its more conservative?  
```

Comment:
Agreed. This should be done based on the AHJ flag.

Reply:
Decided against allowing projects to pass when an economizer is modeled when not required even when AHJ_RA_compare == True.